features/pom.xml

```diff
@@ -89,7 +89,7 @@
         <aries.util.version>1.1.3</aries.util.version>
         <!-- END enterprise properties -->
 
-        <!--START versions for CXF features in platform-app features
+        <!--START versions for CXF features
         Copied from apache-cxf-3.1.13-src/parent/pom.xml
```

Comment:
Should the version here be updated?

Reply:
I should probably remove that piece of the comment. It's not strictly _true_.

Reply:
I would argue to leave the comment and then document exactly what changes were made from the original `/parent/pom.xml` file. I would think that this would help with future upgrades.